osidb/serializer.py

```diff
@@ -796,6 +796,7 @@ class Meta:
                 "cvss3_score",
                 "nvd_cvss3",
                 "is_major_incident",
+                "major_incident_state",
```

Comment:
I am still unsure if this is all that needs to be done.

Reply:
Can you perform an API query filtering flaws with a specific `major_incident_state` value?

Reply:
Yes. If I do `http://localhost:8000/osidb/api/v1/flaws?major_incident_state=REJECTED`, I get `count: 2`.